contentcuration/contentcuration/frontend/channelEdit/components/PublishModal/index.vue

```diff
@@ -165,6 +165,8 @@
       },
     },
     $trs: {
+      publishHeader: 'Publish channel',
```

Comment:
@nucleogenesis Are there any steps required after adding new or updating translations?

Reply:
I think we are fine to introduce new strings - although we will need to work out the timeline for translation and updating, but I'd rather give us a reason to do that than avoid doing it because we haven't worked it out yet.

Reply:
@rtibbles Would it be viable for our own team to translate incoming strings in Studio? I may be wrong but I think our team has people who read & write all three of Studio's non-English languages?